test/framework/resources/templates/vpntunnel-copy.yaml.tpl

```diff
@@ -28,7 +28,7 @@ spec:
         volumeMounts:
         - name: source-data
           mountPath: /data
-      - image: eu.gcr.io/gardener-project/3rd/alpine:3.16.1
+      - image: eu.gcr.io/gardener-project/3rd/alpine:3.16.2
```

Comment:
`eu.gcr.io/gardener-project/3rd/alpine:3.16.2` is not multi-arch. There was a PR from @acumino  that switched all test images to be multi-arch. We don't want to regress this.

Reply:
okay, I did not know that. Changed.